internal/pkg/config/config.go

```diff
@@ -222,6 +246,14 @@ func (s sourceMatcher) match(n *types.Named) bool {
 	return s.PackageRE.MatchString(n.Obj().Pkg().Path()) && s.TypeRE.MatchString(n.Obj().Name())
 }
 
+func (s sourceMatcher) matchFieldName(f *types.Var) bool {
+	return s.FieldRE.MatchString(f.Name())
+}
+
```

Comment:
If the above loop in line 160 is replaced by `Config.IfSourceField`, this method becomes dead.